src/path.ts

```diff
@@ -1,20 +1,23 @@
-import { Optional } from './common';
+import {alwaysTrue, bool, not, Optional} from './common';
 import { head } from './array';
 import { curry } from './curry';
 import { prop } from './prop';
 import { has } from './has';
+import {cond} from "./cond";
+import {compose} from "./compose";
+import {pipe} from "./pipe";
+
+export type StringKeyObject = Record<string, object | string | number | boolean | [] | null >;
```

Comment:
nez to takhle explicitne jmenovat tak bych to radeji vratil na `object: object`

Reply:
oukej, predelano